puble.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>publishing_m</title>
+    <link rel="stylesheet" href="puble.css" />
+  </head>
+  <body>
+    <body class="container">
+      <header class="header">
+        <img src="Group1.png" alt="" class="hamburger-img" ></img>
+        Machiatto
+        <img src="shopping-cart.png" alt="" class="shopping-img" ></img>
+      </header>
+      <div class="icon">
+        <div class="icon-heart">♥</div>
+        <div class="icon-number">256</div>
+      </div>
+<img src="mainCoffee copy.png" alt="" class="main-coffee-img">
+
+<section class="text">     
+<div class="제목">Machiatto</div>
+    <div class="제목글">
+    <br /> Ma Kira To (Machiatto), the espresso add two
+    <br /> tablespoons of milk into a cup of macciato. Macdo
+    <br /> is the brand mark meaning in Italian, so a symbol
+    <br /> of sweet imprint
```

Comment:
br 태그는 / 가 필요 없는 코드입니다.
```suggestion
    <br> Ma Kira To (Machiatto), the espresso add two
    <br> tablespoons of milk into a cup of macciato. Macdo
    <br> is the brand mark meaning in Italian, so a symbol
    <br> of sweet imprint
```